src/pykeen/triples/triples_numeric_literals_factory.py

```diff
@@ -76,13 +96,21 @@ def from_path(
         path: Union[str, pathlib.Path, TextIO],
         *,
         path_to_numeric_triples: Union[None, str, pathlib.Path, TextIO] = None,
+        numeric_triples_preprocessing: Optional[Union[str, Callable[[LabeledTriples], LabeledTriples]]] = None,
+        numeric_literals_preprocessing: Optional[Union[str, Callable[[np.ndarray], np.ndarray]]] = None,
```

Comment:
I'm skeptical about these pre-processing functions since there are only one example of each. Seems like without a variety, these should just be yes/no flags instead of an extensible system that's going to add one more level of complexity for maitenance. Can you do the following:

1. Add additional pre-processsing functions for triples/literals
2. Give specfic examples where they are relevant (both in comment form on this PR and as documentation in code!)

Reply:
The motivation behind adding these arguments is to allow users to conveniently experiment with new preprocessing functions. Also, there exist published models with different preprocessing methods (e.g. the one mentioned in #1207) that need to be easily integrated to pykeen.

Reply:
I added some examples integrated in the docstring of the functions that have these arguments.